### PR TITLE
refacto

### PR DESCRIPTION
**Pull request by Pseudonyme7**

### CODE REVIEW

File: src/app/_shared/landing/landing.component.ts [hunk 1]
```diff
@@ -1,5 +1,5 @@
 import { Component } from '@angular/core';
-import {NavbarComponent} from "../core/layout/navbar/navbar.component";
+import {NavbarComponent} from "../../_teacher/navbar/navbar.component";
```
Comment (fodil-a) [inline src/app/_shared/landing/landing.component.ts:2]: Why _ ?

File: src/app/app-routing.module.ts [hunk 2]
```diff
@@ -15,6 +18,9 @@ export const routerConfig: Routes = [
   { path: 'validation', component: ValidationComponent },
   { path: 'statistiques', component: StatistiquesComponent },
   { path: 'publications', component: PublicationsComponent },
+  { path: 'homestudents', component: HomestudentsComponent },
```
Comment (fodil-a) [inline src/app/app-routing.module.ts:21]: All english or all french but no mix

File: src/app/app.module.ts [hunk 3]
```diff
@@ -5,10 +5,15 @@ import { HttpClientModule } from '@angular/common/http';
...
+import { HomeComponent } from './_teacher/home/home.component';
+import { HomestudentsComponent } from './_students/homestudents/homestudents.component';
+import { FeedstudentsComponent } from './_students/feedstudents/feedstudents.component';
+import { StatsstudentsComponent } from './_students/statsstudents/statsstudents.component';
+import { NavbarstudentsComponent } from './_students/navbarstudents/navbarstudents.component';
 
 
 @NgModule({
-  declarations: [AppComponent],
+  declarations: [AppComponent, HomestudentsComponent, FeedstudentsComponent, StatsstudentsComponent, NavbarstudentsComponent],
```
Comment (fodil-a) [inline src/app/app.module.ts:16]: Formatting

**Review by fodil-a — DISMISSED** [review]
Very small changes fell free to fix before merge

File: src/app/_shared/landing/landing.component.ts (hunk 1 above)
Comment (Pseudonyme7) [reply src/app/_shared/landing/landing.component.ts:2]: Arch was crap had to improve and make some arrangements but i ain't done yet

File: src/app/app-routing.module.ts (hunk 2 above)
Comment (Pseudonyme7) [reply src/app/app-routing.module.ts:21]: alright fam gotta do some more refacto then

File: src/app/app.module.ts (hunk 3 above)
Comment (Pseudonyme7) [reply src/app/app.module.ts:16]: wdym ?

File: src/app/app-routing.module.ts (hunk 2 above)
Comment (Pseudonyme7) [reply src/app/app-routing.module.ts:21]: btw the app was already mixed since there was a "home" route and "statistiques"

Comment (fodil-a) [reply src/app/app-routing.module.ts:21]: maybe just change "Statistiques" 😂

File: src/app/app.module.ts (hunk 3 above)
Comment (fodil-a) [reply src/app/app.module.ts:16]: the line is extra long, difficult to read on my computer/phone
I think there is an issue around formatting the file (prettier ?)

Comment (fodil-a) [reply src/app/app.module.ts:16]: it's extra-minor don't waste time on this

Comment (Pseudonyme7) [reply src/app/app.module.ts:16]: 100 % right gonna fix it right away

**Pull request merged**